### PR TITLE
SDD-884 - Card Template on dashboard

### PR DESCRIPTION
**Pull request by MrKevJoy**
- Changed Card Template on dashboard.

### CODE REVIEW

File: src/Dfc.CourseDirectory.WebV2/Features/ProviderDashboard/Dashboard.cshtml [hunk 1]
```diff
@@ -105,46 +105,49 @@ else if (Model.BulkUploadReadyToGoLiveApprenticeshipsCount > 0)
...
-                            <a asp-controller="BlobStorage" asp-action="GetCurrentCoursesTemplateFile" class="govuk-link">Your published course information (CSV)</a>
-                        </li>
-                        <li>
-                            <a asp-controller="BlobStorage" asp-action="GetBulkUploadTemplateFile" class="govuk-link">A course information template (CSV)</a>
-                        </li>
-                        <li>
-                            <a asp-controller="BulkUpload" asp-action="Courses" asp-all-route-data="@ProviderContext.RouteValues" class="govuk-link">Bulk upload courses</a>
-                        </li>
-                    </ul>
+        @if (FeatureFlagProvider.HaveFeature(FeatureFlags.DataManagement))
```
Comment (gunndabad) [inline src/Dfc.CourseDirectory.WebV2/Features/ProviderDashboard/Dashboard.cshtml:127]: This needs to have an `else` block with the old content in it for when the feature is disabled

File: src/Dfc.CourseDirectory.WebV2/Features/ProviderDashboard/Dashboard.cshtml [hunk 2]
```diff
@@ -292,6 +295,8 @@ else if (Model.BulkUploadReadyToGoLiveApprenticeshipsCount > 0)
         </div>
     </div>
 
+
```
Comment (gunndabad) [inline src/Dfc.CourseDirectory.WebV2/Features/ProviderDashboard/Dashboard.cshtml:298]: Nit: whitespace

File: src/Dfc.CourseDirectory.WebV2/Features/ProviderDashboard/Dashboard.cshtml (hunk 1 above)
Comment (MrKevJoy) [reply src/Dfc.CourseDirectory.WebV2/Features/ProviderDashboard/Dashboard.cshtml:127]: done

File: src/Dfc.CourseDirectory.WebV2/Features/ProviderDashboard/Dashboard.cshtml (hunk 2 above)
Comment (MrKevJoy) [reply src/Dfc.CourseDirectory.WebV2/Features/ProviderDashboard/Dashboard.cshtml:322]: done

**Review by gunndabad — APPROVED** [review]

**Pull request merged**